stylus/cozy-ui/build.styl

```diff
@@ -674,24 +674,13 @@ Display an avatar of a user or contact with initials or an image
     Markup:
     <p class="{{modifier_class}}">I'm visible! 🎉</p>
 
-    .u-hidden - Element is visually hidden but still readable by screen-readers
```

Comment:
Is it a breaking change?

Reply:
No, I forgot to update the doc a long time ago and just noticed today… 😞

Reply:
It was like 8 month ago…